wradlib/io/xarray.py

```diff
@@ -766,19 +766,36 @@ def _preprocess_moment(ds, mom):
     ds = ds.rename({'data': quantity,
                     dims[0]: 'azimuth',
                     dims[1]: 'range'})
+    if mom.parent.decode_coords:
+        coords = mom.parent._get_coords().coords
+        ds = ds.assign_coords(coords)
+        ds = ds.sortby('azimuth')
+        ds = ds.pipe(_reindex_azimuth, mom.parent)
 
     return ds
 
 
 def _reindex_azimuth(ds, sweep, force=False):
+
     dim = list(ds.dims)[0]
     diff = ds[dim].diff(dim)
+
     # this captures missing/additional rays and different angle spacing
     if force | (len(set(diff.values)) > 1):
+        warnings.warn("WRADLIB: missing/additional rays detected "
```

Comment:
Can we silence this warning, after some occurrences? If we load a timeseries of 288 sweeps, and half of the are affected, this will clutter the log...